### PR TITLE
METRICS-5: DAO now uses correct table

### PR DESCRIPTION
**Pull request by Akayeshmantha**
### Description of what I changed
Dao methods not pointing to the correct table.

### Issue I worked on
see :- https://issues.openmrs.org/browse/METRICS-5

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/metrics/api/db/hibernate/HibernateMetricsDAO.java [hunk 1]
```diff
@@ -15,69 +15,66 @@
...
+				.setParameter("endRange", endRange).setParameter("encounterType", encounterType).uniqueResult();
 	}
-	
+
 	@Override
 	public Integer getNewPatientsObjectsByGivenDateRange(LocalDateTime startRange, LocalDateTime endRange) {
-		// @formatter:off
-		String sql = "select count(*) " + "from event_records e " + "where e.title='Patient' and "
-		        + "(e.date_created between :startRange and :endRange) and " + "e.tags = 'CREATED'";
+		// @formatter:Date
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/metrics/api/db/hibernate/HibernateMetricsDAO.java]:
```
// @formatter:off
```

Disables the automatic formatter

```
// @formatter:Date
```

Doesn't really do anything.

File: api/src/main/java/org/openmrs/module/metrics/api/db/hibernate/HibernateMetricsDAO.java [hunk 2]
```diff
@@ -15,69 +15,66 @@
...
+		String sql = "select count(*) " + "from metric_event_records e " + "where e.title='Patient' and "
+				+ "(e.date_created between :startRange and :endRange) and " + "e.tags = 'CREATED'";
 		// @formatter:on
-		
-		return (Integer) sessionFactory.getCurrentSession().createSQLQuery(sql)
-		        .setParameter("startRange", Date.from(startRange.atZone(ZoneId.systemDefault()).toInstant()))
-		        .setParameter("endRange", Date.from(endRange.atZone(ZoneId.systemDefault()).toInstant())).uniqueResult();
+
+		return ((BigInteger) sessionFactory.getCurrentSession().createSQLQuery(sql).setParameter("startRange", startRange)
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/metrics/api/db/hibernate/HibernateMetricsDAO.java:66]: Why `BigInteger` here?

**Review by ibacher — COMMENTED** [review]
Couple of very small things.

File: api/src/main/java/org/openmrs/module/metrics/api/db/hibernate/HibernateMetricsDAO.java (hunk 2 above)
Comment (Akayeshmantha) [reply api/src/main/java/org/openmrs/module/metrics/api/db/hibernate/HibernateMetricsDAO.java:66]: so result is bigint and finally am converting it to the integer value 
```
	return ((BigInteger) sessionFactory.getCurrentSession().createSQLQuery(sql).setParameter("startRange", startRange)
				.setParameter("endRange", endRange).uniqueResult()).intValue();
```

**Comment by Akayeshmantha** [comment]
@ibacher pushed the changes.

**Pull request merged**